src/eeg_publisher/eeg_publisher/mock_publisher.py

```diff
@@ -5,24 +5,37 @@
 from std_msgs.msg import Header
 from eeg_msgs.msg import EEGBlock
 
+
 class MockEEGPublisher(Node):
     def __init__(self):
         super().__init__('mock_eeg_publisher')
-        self.publisher = self.create_publisher(EEGBlock, '/eeg/raw', 10)
-        self.timer = self.create_timer(0.125, self.publish_data)  # 8Hz (32 samples at 256Hz)
-        self.get_logger().info('Mock EEG Publisher running...')
-    
+        np.random.seed(0)
```

Comment:
Maybe like this, so if we want to move some of this to config file, we can have the seed number there as well
```suggestion
         self.n_seed = 0
         np.random.seed(self.n_seed)
```